src/enclave/tls_endpoint.h

```diff
@@ -554,9 +552,9 @@ namespace enclave
 
     // These callbacks below are complex, using the callbacks above and
     // manipulating OpenSSL's BIO objects accordingly. This is just so we can
-    // emulate MbedTLS.
-    // Once we get rid of MbedTLS we can move the callbacks above to handle BIOs
-    // directly and hopefully remove the complexity below.
+    // emulate what MbedTLS used to do.
+    // Now that we have removed it from the code, we can move the callbacks
+    // above to handle BIOs directly and hopefully remove the complexity below.
```

Comment:
Does this deserve its own GitHub issue, or a link to an existing one?

Reply:
This is #3429, should we add that to the comment, too?

Reply:
Yes, that'd be super useful!